### DISCUSSION TITLE
The great Enlightenment? - Or the great "flop"?

### DISCUSSION THREAD

**Discussion by DIrgi** [devemux86/cruiser#593] [Show and tell]
After local tests had been successfully completed and my Garmin has been banned after long years in service, 
on Friday I start my first big tour - 2-3 weeks, 5-6tkm (Germany, Austria, Hungary, Romania, Slovakia & Czechia) with the new cruiser app and my new "Chinese" tablet in a larger navigation task. 
Let's see if everything works fine with a fair budget and scalable solution. Thumbs up.

**Comment by devemux86** [comment]
@DIrgi thank you and please keep us informed about your experience!

**Comment by bios4** [comment]
What were your experiences @DIrgi? 🤔
